pkg/route/registry/route/rest.go

```diff
@@ -103,6 +110,13 @@ func (rs *REST) Update(ctx kapi.Context, obj runtime.Object) (<-chan apiserver.R
 	if errs := validation.ValidateRoute(route); len(errs) > 0 {
 		return nil, errors.NewInvalid("route", route.Name, errs)
 	}
+
+	err := rs.loadCerts(route.TLS)
```

Comment:
I've called this out as needing an update and asked @fabianofranz how it is best implemented in the cli.  https://github.com/openshift/origin/pull/652#issue-54330299  This is certainly not intended to go in as-is since the cert will not be located on the server side